scripts/build/Platform/Linux/deploy_cdk_applications.sh

```diff
@@ -61,6 +61,10 @@ then
     exit 1
 fi
 
+echo [cdk_installation] Install nodejs Node.js v17.x
+curl -fsSL https://deb.nodesource.com/setup_17.x | sudo -E bash -
```

Comment:
I'm not a super big fan of this from a security perspective. We're running a uncontrolled script as root on a headless build node.

Any reason why we don't want to use nvm ( https://github.com/nvm-sh/nvm ), which doesn't require using sudo to install?

Reply:
Looks like someone else brought this up to nodesource's maintainers, but was dismissed: https://github.com/nodesource/distributions/issues/1109

Reply:
I was following the Linux instructions for install nodejs at: https://github.com/nodesource/distributions/blob/master/README.md

I can check whether we can use nvm instead to install nodejs.

Reply:
Thanks, appreciate you taking the feedback